pkg/cache/v3/snapshot.go

```diff
@@ -266,3 +269,30 @@ func (s *Snapshot) ConstructVersionMap() error {
 
 	return nil
 }
+
+// Respond to a delta watch with the provided snapshot value. If the response is nil, there has been no state change.
+func (s *Snapshot) respondDelta(ctx context.Context, request *DeltaRequest, value chan DeltaResponse, state stream.StreamState, log log.Logger) (*RawDeltaResponse, error) {
```

Comment:
Sorry I missed this, can we actually move this func to `simple.go`? It's the simple cache delta response mechanism and even though it uses the snapshot API maybe it makes sense to not make this a method on a Snapshot? SOTW has their `respond()` function separate and localized to the cache

Reply:
ok, moved